db.go

```diff
@@ -534,6 +564,54 @@ func (tx Tx) QueryRowContext(ctx context.Context, query string, args ...interfac
 
 //------------------------------------------------------------------------------
 
+func (tx Tx) Begin() (Tx, error) {
+	return tx.BeginTx(context.Background(), nil)
+}
+
+// BeginTx will save a point in the running transaction.
+func (tx Tx) BeginTx(ctx context.Context, _ *sql.TxOptions) (Tx, error) {
+	uid, err := uuid.NewRandom()
```

Comment:
Let's not add a dependency on `uuid` just to generate a random string.

```go
    b := make([]byte, 16)
    rand.Read(b)
```

Reply:
I used `crypto/rand` but I think also `math/rand` is an option as it is safe for concurrent usage (for this usage pseudo random is fine). Maybe we can check which is faster but that will be one word change.